contracts/v0.8/MarketAPI.sol

```diff
@@ -16,7 +16,7 @@
 //
 // DRAFT!! THIS CODE HAS NOT BEEN AUDITED - USE ONLY FOR PROTOTYPING
 //
-pragma solidity >=0.4.25 <=0.8.15;
+pragma solidity >=0.4.25 <=0.8.17;
```

Comment:
How do we know with which versions it is compatible ? Someone complained the otehr day that he could not compile with 0.8.9 ... Something with Openzepellin.

Reply:
I am actually not sure. I change the version because I needed it in order to use the BigNumber lib. But we are not using it anymore. We should test our contracts with different versions on CI. Maybe something for the future.